pkg/webrtctransport.go

```diff
@@ -222,10 +219,7 @@ func (p *WebRTCTransport) OnICECandidate(f func(c *webrtc.ICECandidate)) {
 
 // OnNegotiationNeeded handler
 func (p *WebRTCTransport) OnNegotiationNeeded(f func()) {
-	debounced := debounce.New(100 * time.Millisecond)
```

Comment:
if we remove this, will we get a negotiate for each track?

Reply:
No, because now we wait unitl all tracks are added to the peer before negotiation is fired.